docs/guidance/stable-diffusion-rayservice.md

```diff
@@ -0,0 +1,55 @@
+# Serve a StableDiffusion text-to-image model using RayService
+
+> **Note:** The Python files for the Ray Serve application and its client are in the [ray-project/serve_config_examples](https://github.com/ray-project/serve_config_examples) repo 
+and [the Ray documentation](https://docs.ray.io/en/latest/serve/tutorials/stable-diffusion.html).
+
+## Step 1: Create a Kubernetes cluster with GPUs
+
+You can follow [aws-eks-gpu-cluster.md](./aws-eks-gpu-cluster.md) to create an AWS EKS cluster with 1
+CPU (`m5.xlarge`) and 1 GPU (`g5.xlarge`) nodes.
+
+## Step 2: Install KubeRay operator
+
+Follow [this document](../../helm-chart/kuberay-operator/README.md) to install the nightly KubeRay operator via 
+Helm. Note that the YAML file in Step 3 uses `serveConfigV2`, which is first supported by KubeRay v0.6.0.
+
+## Step 3: Install a RayService
+
+```sh
+# path: ray-operator/config/samples/
+kubectl apply -f ray-service.stable-diffusion.yaml
+```
+
+* The `tolerations` for workers must match the taints on the GPU node group. Without the tolerations, worker Pods won't be scheduled on GPU nodes.
+    ```yaml
+    # Please add the following taints to the GPU node.
+    tolerations:
+        - key: "ray.io/node-type"
+        operator: "Equal"
+        value: "worker"
+        effect: "NoSchedule"
+    ```
+* Install `diffusers` in `runtime_env` as it is not included by default in the `ray-ml` image.
```

Comment:
Can you add a code snippet for this recommendation, so users can copy and paste it?

Reply:
Users do not need to install `diffusers` themselves. The YAML file has already configured the runtime_env.

<img width="971" alt="Screen Shot 2023-06-22 at 9 26 38 PM" src="https://github.com/ray-project/kuberay/assets/20109646/6e19ebec-719f-4f95-9f3a-18b45a5b2cc5">

Reply:
```suggestion
* It includes `diffusers` in `runtime_env` since this package is not included by default in the `ray-ml` image.
```